solr/test-framework/src/java/org/apache/solr/SolrTestCase.java

```diff
@@ -183,4 +186,11 @@ public void checkSyspropForceBeforeAssumptionFailure() {
     final String PROP = "tests.force.assumption.failure.before";
     assumeFalse(PROP + " == true", systemPropertyAsBoolean(PROP, false));
   }
+
+  public static void jsonAssert(String expected, String actual) {
```

Comment:
Without looking at the code, the method's name alone gives no hint as to what is being asserted.  Perhaps renaming to assertJsonEquals instead?  And javadocs of course.